test/container_test.py

```diff
@@ -372,6 +372,18 @@ def test_handles_sbs384_containers(self, dummy_384):
             dummy_384.wells_from_shape(0, SHAPE(rows=8, columns=12))
             == dummy_384.quadrant(0)
         )
+        assert(
```

Comment:
These new added tests were failing with the code on the current master. The fix made these tests passing.